File: manual/java-src/com/atta/mcpanel/overlay/OverlayReceiver.java
package com.atta.mcpanel.overlay;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

/** دریافت دکمهٔ «توقف پنل» از اعلان */
public class OverlayReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        OverlayService.stop(context);
    }
}

File: app/src/main/java/com/atta/mcpanel/overlay/OverlayReceiver.kt
package com.atta.mcpanel.overlay

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

/** دریافت دکمهٔ «توقف پنل» از اعلان. */
class OverlayReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        if (intent.action == ACTION_STOP) {
            OverlayService.stop(context)
        }
    }

    companion object {
        const val ACTION_STOP = "com.atta.mcpanel.action.STOP_OVERLAY"
    }
}
